packages/roosterjs-content-model/lib/publicTypes/format/formatParts/BorderFormat.ts

```diff
@@ -21,4 +21,24 @@ export type BorderFormat = {
      * Left border in format 'width style color'
      */
     borderLeft?: string;
+
+    /**
+     * Color to be applied in all borders
+     */
+    borderColor?: string;
+
+    /**
+     * Width to be applied in all borders
+     */
+    borderWidth?: string;
+
+    /**
+     * Style to be applied in all borders
+     */
+    borderStyle?: string;
```

Comment:
These 3 will be conflict with borderLeft/Top/Right/Bottom. We just need to keep one set of them.